assets/src/block-templates/quick-links/template.js

```diff
@@ -0,0 +1,43 @@
+import mainThemeUrl from '../main-theme-url';
+
+const {__} = wp.i18n;
+
+const category = ['core/column', {}, [
+  ['core/group', {className: 'group-stretched-link'}, [
+    ['core/image', {
+      align: 'center',
+      className: 'is-style-rounded-90 force-no-lightbox force-no-caption mb-0',
+      url: `${mainThemeUrl}/images/placeholders/placeholder-90x90.jpg`,
```

Comment:
If you have time you can try to fix the current console warning complaining that save function and post body don't have the same output, I think by adding width and height definition `width:90,height:90,`